src/inputs/SearchBar/SearchBar.js

```diff
@@ -4,7 +4,7 @@ import React, { useRef, useState } from 'react';
 import PropTypes from 'prop-types';
 import { TextField, InputAdornment } from '@mui/material';
 
-export const SearchBar = ({ onSearchChange, className, label, icon, id }) => {
+export const SearchBar = ({ onSearchChange, className, label, icon, id, ...other }) => {
```

Comment:
Maybe _others_ instead _other_ ?

Reply:
In other components that follow the same pattern, we've named it `other` or `otherProps`. I'll change it to `otherProps` :slightly_smiling_face: